src/run/runner/wall_time/executor.rs

```diff
@@ -26,6 +26,11 @@ impl WallTimeExecutor {
     fn walltime_bench_cmd(config: &Config, run_data: &RunData) -> Result<String> {
         let bench_cmd = get_bench_command(config)?;
 
+        // We have to forward all the environment variables.
```

Comment:
I fear that we'll end up with unwanted information in the command, which is written to our runner.log and uploaded on our servers. Am I overthinking this ?

Reply:
Systemd has EnvironmentFile so adding `--property=EnvironmentFile={env_file_path}` should work